extensions/tmtv/src/utils/measurementServiceMappings/utils/getSOPInstanceAttributes.js

```diff
@@ -0,0 +1,69 @@
+import * as cornerstone from '@cornerstonejs/core';
+
+/**
+ * It checks if the imageId is provided then it uses it to query
+ * the metadata and get the SOPInstanceUID, SeriesInstanceUID and StudyInstanceUID.
+ * If the imageId is not provided then it uses the sceneUID to get the viewports
+ * inside the scene and then it checks each viewport to find the one that has
+ * acquisition plane view, and uses the currentImageId of the viewport to
+ * query the metadata and get UIDs.
+ * @param {string} imageId The image id of the referenced image
+ * @param {string} sceneUID The scene UID of the measurement tool
+ * @returns
+ */
+export default function getSOPInstanceAttributes(
+  imageId,
+  Cornerstone3DViewportService,
```

Comment:
Wouldn't it be a more natural api to pass in the servicesManager, and then attempt to get this service (throwing if it does not exist)? (I know this isn't used yet, but we could at least change the API ahead of time, right now this is a mandatory argument even though it isn't used).